src/CalcViewModel/DataLoaders/DataLoaderMockConstants.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+namespace CalculatorApp
+{
+    namespace DataLoaders
+    {
+        static constexpr auto sc_MetadataUriLocalizeFor = L"https://go.microsoft.com/fwlink/?linkid=2091028&localizeFor=";
```

Comment:
nit: I am a _little_ confused why Venus' currency is "Earth Rocks".  It seems a bit odd for a civilization to name their currency after a _different_ civilization...or is it simply that they also use "Earth" to indicate the ground?

Reply:
Their favorite planet is earth, so earth rocks are pretty popular there... but apparently they also use Venusian Seashells according to @EriWong so I have updated the data.